### PR TITLE
add job roles UI table

### PR DESCRIPTION
**Pull request by maciekkaczor**

### CODE REVIEW

File: app/jobRoleData.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const axios = require('axios')
+
+exports.getJobRoles = async () => { 
+    let jobRoles = []  
+      try {  
+        const jobRoleResponse = await axios.get('http://localhost:8080/api/job-roles')
+        jobRoles = jobRoleResponse.data
```
Comment (shaunganley) [inline app/jobRoleData.js]: you can just `return jobRoleResponse.data` rather than at the end

File: app/views/jobRolesView.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    head: [ { text: "Job Role" }], 
+    rows: jobRolesrows }) 
+}} 
+{% else %}
+There are no Job Roles. 
+{% endif %}
+
+{{ govukButton({
+  text: "Home",
+  href: "http://localhost:3000/home"
```
Comment (shaunganley) [inline app/views/jobRolesView.html]: including the localhost part will cause issues when this is deployed to the cloud

**Review by shaunganley — CHANGES_REQUESTED** [review]
small comments, good work

**Review by shaunganley — APPROVED** [review]

File: app/views/layout.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+{% from "govuk/components/tag/macro.njk"                 import govukTag %}
+{% from "govuk/components/textarea/macro.njk"            import govukTextarea %}
+{% from "govuk/components/warning-text/macro.njk"        import govukWarningText %}
+
+{% block head %}
+  {% include "includes/head.html" %}
+{% endblock %}
+
+{% block pageTitle %}
+  GOV.UK Prototype Kit
```
Comment (boydstepnew) [inline app/views/layout.html:39]: Can we change this?

File: app/views/jobRolesView.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% extends "layout_unbranded.html" %}
+
+{% block pageTitle %}
+  GOV.UK page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (boydstepnew) [inline app/views/jobRolesView.html]: Let's remove the references to the template

File: app/jobRoleData.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+const axios = require('axios')
+
+exports.getJobRoles = async () => { 
+      try {  
+        const jobRoleResponse = await axios.get('http://localhost:8080/api/job-roles')
+        return jobRoleResponse.data
+      } catch (e) {
+         return new Error('Could not get job roles')
```
Comment (boydstepnew) [inline app/jobRoleData.js]: Does this exception contain a message we can also add to the error message? The more detailed we can be the better

File: app/views/jobRolesView.html (hunk 4 above)
Comment (maciekkaczor) [reply app/views/jobRolesView.html]: fixed

**Review by boydstepnew — APPROVED** [review]

**Review by boydstepnew — APPROVED** [review]

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**